packages/react-core/src/components/LoginPage/LoginMainHeader.tsx

```diff
@@ -12,13 +12,16 @@ export interface LoginMainHeaderProps extends React.HTMLProps<HTMLDivElement> {
   title?: string;
   /** Subtitle that contains the Text, URL, and URL Text for the Login Main Header */
   subtitle?: string;
+  /** Select menu that renders next to the Login Title for the Login Main Header */
```

Comment:
nit - FWIW this could contain anything (likely actions of some sort), and it isn't always by the title, it's below the subtitle on small screens. I wonder if we could just describe this as a place for actions, such as a language selector, or something like that?